server/test_helpers.go

```diff
@@ -42,27 +39,27 @@ func setupViper(t *testing.T) func() {
 
 // Begin the server pass up the channel to close
 // NOTE pass up the channel so it can be closed at the end of the process
-func StartServer(t *testing.T) chan error {
-	defer setupViper(t)()
+//func StartServer(t *testing.T, cdc *wire.Codec) chan error {
```

Comment:
Are we no longer using this - if so, any reason not to just delete it?

Always will be stored in Git history, IMO not necessary to keep commented code around unless we explicitly intend to re-enable it later.

Reply:
just noticed it wasn't used so I commented it out, concerned that somebody else might want it but then again they should just go to the git version history need be

Reply:
Addressed